### DISCUSSION TITLE
Database Selection

### DISCUSSION THREAD

**Discussion by JimmyTron** [SpaceyaTech/Team-Rio-Django#14] [Ideas]
Choosing a database for the project

**Comment by sangkips** [comment]
Which database do we use with this project? PostgreSQL or MySQL?

**Comment by JimmyTron** [comment]
For the database we'll use postgresql since we are looking to have a scallable project.

**Comment by JimmyTron** [comment]
who is taking up the task to create the dockerised container for the database(postgresql)?

**Comment by JosephThuku** [comment]
i think postgresql is cool

**Comment by JimmyTron** [comment]
![image](https://user-images.githubusercontent.com/23496280/209255800-ab002dc5-07ec-4ad6-a5c0-f79471d7b821.png)

[Postgres Notes](https://docs.djangoproject.com/en/4.1/ref/databases/#postgresql-notes)

**Reply by JimmyTron** [reply]
![image](https://user-images.githubusercontent.com/23496280/209256203-52fefbcb-efb4-48af-a699-a578c59fff3f.png)
